src/entities/User.ts

```diff
@@ -66,9 +68,13 @@ export class User {
 	@JoinColumn()
 	public profile?: Profile;
 
+	@OneToOne(() => Report, report => report.reportedUser, { nullable: true, eager: true, cascade: true })
+	@JoinColumn()
+	public report?: Report;
```

Comment:
Can you update this to be a one-to-many, and remove the options? I don't think it's necessary to fetch this by default.